SecondDeliverable/controller/csvcontroller.go

```diff
@@ -0,0 +1,98 @@
+package controller
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"main/usecase"
+
+	"github.com/gorilla/mux"
+)
+
+// Index ..
+func Index(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Welcome to my Diablo 3 Items-API")
+}
+
+<<<<<<< Updated upstream
+// GetItems ..
+=======
+// GetItems - Get items from CSV
+>>>>>>> Stashed changes
+func GetItems(w http.ResponseWriter, r *http.Request) {
+	items, err := usecase.GetItems()
+	w.Header().Set("Content-Type", "application/json")
+	if err != nil {
+		w.WriteHeader(err.Code)
+		fmt.Fprint(w, "Something happened..%v", err.Message)
+	}
+	json.NewEncoder(w).Encode(items)
+}
+
+// GetItem - Get an item from CSV
+func GetItem(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	itemID := vars["id"]
+	if itemID == "" {
+		fmt.Fprint(w, "Invalid ID")
```

Comment:
This is a path param, so the endpoint needs this param, to be a valid endpoint. 
I think there is no way to have an empty path param, because  the router will respond with Not Found